guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -134,12 +134,14 @@ learn about Ruby on Rails, and the API, which serves as a reference.
 
 You can help improve the Rails guides by making them more coherent, consistent or readable, adding missing information, correcting factual errors, fixing typos, or bringing them up to date with the latest edge Rails.
 
-To do so, open a pull request to [Rails](https://github.com/rails/rails) on GitHub.
+To do so, open a pull request to [Rails](https://github.com/rails/rails) on GitHub. Source files for Rails guides are located [here](https://github.com/rails/rails/tree/master/guides/source).
 
 When working with documentation, please take into account the [API Documentation Guidelines](api_documentation_guidelines.html) and the [Ruby on Rails Guides Guidelines](ruby_on_rails_guides_guidelines.html).
 
 NOTE: To help our CI servers you should add [ci skip] to your documentation commit message to skip build on that commit. Please remember to use it for commits containing only documentation changes.
 
+NOTE2: In case you are new to GitHub, you need to register and then make an edit to Rails guide source files. Upon that a new fork will be created, which you can use in new pull request.
```

Comment:
On one hand I want the guides to be as accessible as possible, on the other hand Rails shouldn't include documentation on _how_ to use GitHub. Docs on how to use GitHub is GitHub's job, so I'm leaning towards removing this.

Reply:
The thing is that the instruction to create pull request is confusing since you can't make pull request unless you create a new branch and change the source code for documentation. Maybe it was just me being unattentive, but still, there can probably be more newcomers like me.

Ok, here is a new variant. Without "note" section, that just says that to contribute one needs to edit source files and **then** make a pull request.
[See this commit](https://github.com/rails/rails/pull/31704/commits/3bf3863b3f0a2f5ac0215db4b9bd94ddb6d458d2)